blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/OrganizationFolderPipelineImpl.java

```diff
@@ -33,6 +32,11 @@ public OrganizationFolderPipelineImpl(OrganizationFolder folder, Link parent) {
 
     }
 
+    @Exported
+    public String getIcon() {
+        return folder.getIcon().getImageOf("32x32");
```

Comment:
what icon is it? might be ok to fix this up in another PR (if it is the generic github icon that is fine, as most peoples icons are horrendous anyway)